src/BookmarksPanel.svelte

```diff
@@ -32,11 +36,33 @@
         anchor.click();
         URL.revokeObjectURL(url);
     }
+
+    function updateColorSelection() {
+        // Toggle whether or not to change/export node colors
+        let cb = document.getElementById("color-toggle");
+
+        if (cb.checked === true) {
+            document.getElementById("color-selection").disabled = false;
+            colorFlag = true;
+        }
+        else {
+            document.getElementById("color-selection").disabled = true;
+            colorFlag = false;
+        }
+    }
 </script>
 
 <BackButton on:back={() => dispatch("showSearch")} />
 <h2>ATT&CK Powered Suit</h2>
 <h3><i class="bi bi-bookmark-fill" /> Bookmarks</h3>
+<h3>
+    <input
+        type="checkbox"
+        id="color-toggle"
+        on:click={() => updateColorSelection()}
```

Comment:
This can be simplified using Svelte binding. Instead of `on:click=...`, use `bind:checked={colorFlag}`. Then you can delete the body of `updateColorSelection()`..